templates/zerver/emails/email.css

```diff
@@ -20,19 +21,20 @@ body {
     -webkit-text-size-adjust: 100%;
 }
 
-pre {
-    font-family: sans-serif;
-    font-size: 14px;
-}
-
-table {
+table.body,
+table.main,
+td.wrapper > table,
+.footer > table {
     border-collapse: separate;
     mso-table-lspace: 0;
     mso-table-rspace: 0;
     width: 100%;
 }
```

Comment:
I think this block would be much better done with demantic styles -- e.g. attaching a `full-width` class and calling this something like `table.email-layout-table`. I'm not sure if that's a good name for this though.

Reply:
I'm not sure if I understood your thought correctly, but I made some changes that feel good, inspired by your comment.